src/server/src/main/java/io/cassandrareaper/service/SegmentRunner.java

```diff
@@ -675,7 +675,9 @@ public void handle(
         } finally {
           // if someone else does hold the lease, ie renewLead(..) was true,
           // then their writes to repair_run table and any call to releaseLead(..) will throw an exception
-          releaseLead();
+          try {
+            releaseLead();
+          } catch (AssertionError ignore) { }
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Either remove or fill this block of code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00108)